src/printer.js

```diff
@@ -98,8 +98,7 @@ function genericPrint(path, options, printPath, args) {
 
       if (
         node.decorators.length === 1 &&
-        (decorator.type === "Identifier" ||
```

Comment:
This is a breaking change. Aren't single identifier decorators supposed to be inlined in mobx?